src/less/mixins/layout/layout-mixins.less

```diff
@@ -0,0 +1,4 @@
+.flex-items-centered() {
+    align-items: center;
```

Comment:
This also needs a `display: flex` or `display: inline-flex` to work. I see it is added in the other parts of the code, but someone using this might forget to add the correct display. 
Maybe we should add an argument to set the display to flex and if the argument for this mixin is null then it will default to `flex` or something. That way this is a standalone mixin and doesn't require any display properties outside of it. 
IE:
```
.flex-items-centered(@display: flex) {
   display: @display;
   // Other props. 
}
```
And then call it like:
`.flex-items-centered(inline-flex)`

Reply:
Yep, I like that idea. Will update.

Reply:
Done.